manipulate_pim_data/measurement/configure_measurement_limits.rst

```diff
@@ -1,43 +0,0 @@
-How to Configure Measurement Limits
```

Comment:
This documentation is always maintain it have been added on 5.0 after measurement release